### ISSUE TITLE
Examples?

### ISSUE THREAD

**Issue by thealjey** [tangkunyin/typedoc-plugin-toc-group#4]
I've tried adding a `@group` tag to my files, with the theme installed, and literally nothing happened.

**Comment by tangkunyin** [comment]
@thealjey Did you add the tag like this: `@group title` ?
If you add tag without `title` then there is nothing happened finally. 🤣

**Comment by thealjey** [comment]
@tangkunyin of course. It shows up in `dl.tsd-comment-tags`, but then, the exact same thing happens with the default theme and without the plugin.
The navigation and/or the TOC are not affected at all.

**Comment by airtonix** [comment]
yep looks likes this plugin isn't compatiable with the latest release of typedoc...

which is a shame, since it does try to solve a somewhat important issue of presentation.

**Comment by airtonix** [comment]
On reflection, I'm not convinced this is needed anymore.

You can either use this theme to control the sidebar: https://github.com/google/typedoc-neo-theme#outline

or you can use the default theme and mark your various comments with `@category Any Phrase you want here`
